### PR TITLE
Improve release....

### PR DESCRIPTION
**Pull request by brendandburns**
Looks like I wasn't doing a clean build for a release, which caused #258 

This improves the build script to delete any pre-existing files.

### CODE REVIEW

File: build-package.sh [hunk 1]
```diff
@@ -3,7 +3,17 @@
 # exit on any error
 set -e
 
+# pre-requisites
+npm run format
+npm test
+
+# validate branches
```
Comment (drubin) [inline build-package.sh]: I would personally move this above any of the other checks.

Comment (brendandburns) [reply build-package.sh]: I was actually using this to make sure that format didn't change anything...

e.g. if the `npm run format` call modifies code then the validation below will fail.

**Comment by brendandburns** [comment]
Comments addressed, please take another look.

**Comment by drubin** [comment]
Checked it out and ran it locally 

/lgtm

**Pull request merged**